### DISCUSSION TITLE
请问如何在一台vps同时启动两个CQPS

### DISCUSSION THREAD

**Discussion by ghost** [Tsuk1ko/CQ-picfinder-robot#153] [问题求助] [answered]
有两个单独的bot均需要使用cqps的搜图服务，请问我该如何在同一台vps上启动两个cqps
目前，不论我先启动哪一个cqps，启动第二个cqps时一定是重启第一个cqps而不是启动第二个

环境Centos 7 node 14.16.0 pm2 4.5.0

**Comment by Tsuk1ko** [comment, marked as answer]
试试将另一个 cqps 的 pm2 配置文件中的 name 改掉

https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/ff1c1c4bd074bbab74dbe881c14d7a949cda51d9/ecosystem.config.js#L4

**Reply by ghost** [reply]
> 试试将另一个 cqps 的 pm2 配置文件中的 name 改掉
> 
> https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/ff1c1c4bd074bbab74dbe881c14d7a949cda51d9/ecosystem.config.js#L4

谢谢神代佬，但是我遇到了新的问题提示找不到config.jsonc

![jietu](https://i.loli.net/2021/03/21/ezF7jMSydK9XspH.png)

请问这可能是什么造成的

**Reply by Tsuk1ko** [reply]
`config.default.jsonc` 需要保留，不能直接将其改名

**Reply by ghost** [reply]
> `config.default.jsonc` 需要保留，不能直接将其改名

谢谢神代佬，解决了，另外是不是移除正则表达式中(?<r18>[Rr]18的?)就可以禁止r18了

**Reply by Tsuk1ko** [reply]
是，不过其实默认 `r18OnlyInWhite` 是 `true`，只有白名单群组才能 r18

**Reply by ghost** [reply]
> 是，不过其实默认 `r18OnlyInWhite` 是 `true`，只有白名单群组才能 r18

谢谢

**Reply by Miuzarte** [reply]
> > `config.default.jsonc` 需要保留，不能直接将其改名
> 
> 谢谢神代佬，解决了，另外是不是移除正则表达式中(?[Rr]18的?)就可以禁止r18了

可以直接在下面自动回复里写上个`^r18.*色图来$`,顺便还能单独分开群聊和私聊
